<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportItemsDialog</class>
 <widget class="QDialog" name="ImportItemsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Items</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="file_path_layout">
     <item>
      <widget class="QLabel" name="file_path_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>File Path:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="file_path_entry">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="file_path_button">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tab_properties">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="columns_tab">
      <attribute name="title">
       <string>Columns</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="name_layout">
         <item>
          <widget class="QLabel" name="name_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="name_spin">
           <property name="specialValueText">
            <string>None</string>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="chem_formula_layout">
         <item>
          <widget class="QLabel" name="chem_formula_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Chemical Formula:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="chem_formula_spin">
           <property name="specialValueText">
            <string>None</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="warning_label_layout">
         <item>
          <widget class="QLabel" name="warning_label_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Warning Label:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="warning_label_spin">
           <property name="specialValueText">
            <string>None</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="danger_level_layout">
         <item>
          <widget class="QLabel" name="danger_level_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Danger Level:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="danger_level_spin">
           <property name="specialValueText">
            <string>None</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="notes_layout">
         <item>
          <widget class="QLabel" name="notes_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Notes:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="notes_spin">
           <property name="specialValueText">
            <string>None</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="import_button">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>36</height>
      </size>
     </property>
     <property name="text">
      <string>Import Items</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
